ioctl/newcmd/node/nodeprobationlist.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2022 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package node
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/iotexproject/iotex-core/ioctl"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/newcmd/bc"
+)
+
+// Multi-language support
+var (
+	_probationlistCmdUses = map[config.Language]string{
+		config.English: "probationlist [-e epoch-num]",
+		config.Chinese: "probationlist [-e epoch数]",
+	}
+	_probationlistCmdShorts = map[config.Language]string{
+		config.English: "Print probation list at given epoch",
+		config.Chinese: "",
+	}
+)
+
+// NewNodeProbationlistCmd represents querying probation list command
+func NewNodeProbationlistCmd(c ioctl.Client) *cobra.Command {
+	var epochNum uint64
+	use, _ := c.SelectTranslation(_probationlistCmdUses)
+	short, _ := c.SelectTranslation(_probationlistCmdShorts)
+	flagEpochNumUsages, _ := c.SelectTranslation(_flagEpochNumUsages)
+
+	cmd := &cobra.Command{
+		Use:   use,
+		Short: short,
+		Args:  cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
+			if epochNum == 0 {
+				chainMeta, err := bc.GetChainMeta(c)
+				if err != nil {
+					return errors.New("failed to get chain meta")
+				}
+				epochNum = chainMeta.Epoch.Num
```

Comment:
use `chainMeta.GetEpoch()` to check nil pointer . see  https://github.com/iotexproject/iotex-core/blob/bf98850736839425497ac336fee828db4961d617/ioctl/cmd/node/nodeprobationlist.go#L72-L73